server/paymentProviders/privacy/index.ts

```diff
@@ -90,7 +90,7 @@ const resumeCard = async (virtualCard: VirtualCardModel): Promise<VirtualCardMod
   setCardState(virtualCard, 'OPEN');
 
 const deleteCard = async (virtualCard: VirtualCardModel): Promise<void> => {
-  const host = await models.Collective.findByPk(virtualCard.HostCollectiveId);
+  const host = virtualCard.host;
```

Comment:
Not a requested change.

Just to note that I think this is more fragile for the future.

One day, a developer will reuse this and pass a `virtualCard` without the `host` relation, this will fail.

It's a bit difficult generally to keep track of the relations required by a function, so more practicable to not rely on them and request them on demand,